x/bep3/abci.go

```diff
@@ -0,0 +1,14 @@
+package bep3
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+// BeginBlocker runs at the start of every block
+func BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock, k Keeper) {
+	err := k.UpdateExpiredAtomicSwaps(ctx)
+	if err != nil {
```

Comment:
To match bnb implementation, we can add method to delete  expired swaps after 2 weeks. Can be in follow on PR.

Reply:
Issue https://github.com/Kava-Labs/kava/issues/384